src/check-dapi-name.ts

```diff
@@ -0,0 +1,60 @@
+import { ethers } from 'ethers';
+import { readOperationsRepository } from './utils/read-operations';
+import { runAndHandleErrors } from './utils/cli';
+import { chainNameToChainId, DapiServerContract } from './utils/evm';
+import { loadCredentials } from './utils/filesystem';
+
+const main = async () => {
+  const credentials = loadCredentials();
+  const operationsRepository = readOperationsRepository();
+
+  const DapiNamePromises = Object.entries(operationsRepository.dapis).flatMap(([chainName, dAPI]) =>
+    Object.entries(dAPI).map(async ([dAPIName, dataFeedId]) => {
```

Comment:
Should we rename `dataFeedId` to `resourceId` after today's discussion? 🤭

Reply:
I don't think so, each dAPIName maps to a `dataFeedId` but a resouceId can refer to a dAPIName or a dataFeedId . which is why I don't like the concept of a resourceId. it just adds confusion